src/server_manager/install_scripts/install_server.sh

```diff
@@ -377,8 +396,12 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
   PUBLIC_HOSTNAME=${FLAGS_HOSTNAME:-${SB_PUBLIC_IP:-$(curl -4s https://ipinfo.io/ip)}}
+  local readonly validation=`validate_ipv4 ${PUBLIC_HOSTNAME}`
```

Comment:
Use $() instead of backticks: https://engdoc.corp.google.com/eng/doc/shell.xml?showone=Command_Substitution&cl=head#Command_Substitution

Reply:
No longer relevant